website/docs/docs/supported-data-platforms.md

```diff
@@ -20,24 +20,7 @@ import MSCallout from '/snippets/_microsoft-adapters-soon.md';
 
 ## Types of Adapters
 
-There are three types of adapters available today:
+There are two types of adapters available today:
 
-- **Verified** &mdash; [Verified adapters](verified-adapters) are those that have completed a rigorous verification process in collaboration with dbt Labs.
-- **Trusted** &mdash; [Trusted adapters](trusted-adapters) are those where the adapter maintainers have agreed to meet a higher standard of quality.
+- **Trusted** &mdash; [Trusted adapters](trusted-adapters) are those where the adapter maintainers have agreed to meet a higher standard of quality. For adapters supported in dbt Cloud, maintainers have gone through an additional rigorous process that cover contractual requirements for development, documentation, user experience, and maintenance.
```

Comment:
Maybe now is a good time to reword this so as not to appear to be throwing shade at community adapters
```suggestion
- **Trusted** &mdash; [Trusted adapters](trusted-adapters) are those where the adapter maintainers have undergone a rigorous process that covers contractual requirements for development, documentation, user experience, and maintenance.
```

Reply:
This is actually slightly incorrect (but also happy to avoid throwing shade :)). 

- Trusted adapters do not have any contractual requirements. They have opted into the program and we are going honesty policy to say they have met the requirements. If they're part of dbt Cloud - they do have contractual requirements. 
How do we write that?

Reply:
@amychen1776 : drafted something for this section. lemme know what you think and i'll update the other one too with your thoughts folded in